docs/user-guide-102-configuration.md

```diff
@@ -0,0 +1,443 @@
+# Introduction: Configuration
+
+The operator allows you to configure various network interface types, DNS and
+routing on cluster nodes. The configuration is driven by two main object types,
+`NodeNetworkConfigurationPolicy` (Policy) and
+`NodeNetworkConfigurationEnactment` (Enactment).
+
+A Policy describes what is the desired network configuration on cluster nodes.
+It is created by used and applies cluster-wide. On the other hand, an Enactment
+is a read-only object that carries execution state of a Policy per each Node.
+
+Policies are applied on node via NetworkManager. Thanks to this, the
+configuration is persistent on the node and survives reboots.
+
+## Creating interfaces
+
+Each Policy has a name (`metadata.name`) and desired state
+(`spec.desiredState`). The desired state can contains declarative specification
+of the node network configuration following [nmstate
+API](https://nmstate.github.io/).
+
+In the following example, we will create a Policy that configures a bonding over
+NICs `eth1` and `eth2` across the whole cluster.
+
+First of all, let's apply the manifest:
+
+<!-- When updating following example, don't forget to update respective attached file -->
+
+[Download example](examples/bond0-eth1-eth2_up.yaml)
+
+```yaml
+apiVersion: nmstate.io/v1alpha1
+kind: NodeNetworkConfigurationPolicy
+metadata:
+  name: bond0-eth1-eth2
+spec:
+  desiredState:
+    interfaces:
+    - name: bond0
+      type: bond
+      state: up
+      ipv4:
+        dhcp: true
+        enabled: true
+      link-aggregation:
+        mode: balance-rr
+        slaves:
+        - eth1
+        - eth2
+```
+
+```shell
+kubectl apply -f bond0-eth1-eth2_up.yaml
+```
+
+Wait for the Policy to be successfully applied:
+
+```shell
+kubectl wait nncp bond0-eth1-eth2 --for condition=Available --timeout 2m
+```
+
+List all the Policies applied on the cluster:
+
+```shell
+kubectl get nodenetworkconfigurationpolicies
+```
+
+```
+NAME              STATUS
+bond0-eth1-eth2   SuccessfullyConfigured
+```
+
+You can also use short name `nncp` to reach the same effect:
+
+```shell
+kubectl get nncp
+```
+
+```
+NAME              STATUS
+bond0-eth1-eth2   SuccessfullyConfigured
+```
+
+By using `-o yaml` you obtain the full Policy with its current state:
+
+```shell
+kubectl get nncp bond0-eth1-eth2 -o yaml
```

Comment:
Maybe we can do -o=jsonpath={.status} so we don't have to say nothing about runcate

Reply:
That returns JSON, so it is impossible to read in this size